### ISSUE TITLE
veh debugger doesn't work if manually injected

### ISSUE THREAD

**Issue by AlizerUncaged** [cheat-engine/cheat-engine#2571]
I have a x64 game that seems to be blocking .dll injection via LoadLibrary, so I manually mapped ``vehdebug-x86_64.dll`` into the game (manually mapping the dll files work), however when I attempt to run "Find out what writes to this address" when right clicking an address it says I get the following error

![image](https://user-images.githubusercontent.com/86959368/236663553-f8276a4a-e028-46f8-9b81-23c5e3f02182.png)

is it not possible to manually inject the veh debugger dll?

**Comment by cheat-engine** [comment]
yes, you just need to register the symbol for "vehdebugxxx.InitializeVEH" and make sure that the entrypoint of the dll is executed once
